x-pack/plugins/security_solution/public/common/components/import_data_modal/translations.ts

```diff
@@ -40,3 +45,29 @@ export const IMPORT_FAILED = (totalExceptions: number) =>
         'Failed to import {totalExceptions} {totalExceptions, plural, =1 {exception} other {exceptions}}',
     }
   );
+
+export const ACTION_CONNECTORS_WARNING_TITLE = (totalConnectors: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.components.importRuleModal.actionConnectorsWarningTitle',
+    {
+      values: { totalConnectors },
+      defaultMessage: '{totalConnectors} connector imported',
```

Comment:
Pluralization for `connectors`:

<img width="308" alt="image" src="https://user-images.githubusercontent.com/2946766/215904788-0738b146-0d5e-429f-9600-7a428dfff6cc.png">


```suggestion
      defaultMessage: '{totalConnectors} {totalConnectors, plural, =1 {connector} other {connectors}} imported',
```

Reply:
please review 2adf31df63980d215ae173b27e3db8bf07ddfb2c